Tests/SonarScanner.MSBuild.Common.Test/CmdLineArgsPropertiesProviderTests.cs

```diff
@@ -141,18 +141,22 @@ public void CmdLineArgProperties_Disallowed_DynamicProperties()
             logger.AssertSingleErrorExists(SonarProperties.WorkingDirectory);
         }
 
-        [TestMethod]
-        [Description("Test for https://jira.sonarsource.com/browse/SONARMSBRU-208")]
-        public void SonarProjectBaseDir_IsAllowed()
+        [DataTestMethod]
+        [DataRow("sonar.projectBaseDir=value1")] // Test for https://jira.sonarsource.com/browse/SONARMSBRU-208
+        [DataRow("sonar.scanner.sonarcloudUrl=value1")]
+        [DataRow($"{SonarProperties.SonarcloudUrl}=value1")]
```

Comment:
I have mixed feelings about defining here the same property twice, in different ways. I'm not sure what value it brings. Are you concerned that we will rename the property?

Reply:
The sut doesn't really care about the names (with a few exceptions). Therefore the duplication does not really achieve anything and I removed one instance.